### PR TITLE
update 'top' api endpoint documentation

### LINKED ISSUE #7205: top api is not consistent
Hi,
I am quite new to docker, but already impressed about its capability!

One question I have is when I run the /container/id/top api, I got some attributes value back, but not as the ones I see in documentation.

From documentation, I see:
List processes running inside a container
GET /containers/(id)/top

List processes running inside the container id

**Example request**:

    GET /containers/4fa6e0f0c678/top HTTP/1.1

**Example response**:

    HTTP/1.1 200 OK
    Content-Type: application/json

    {
         "Titles":[
                 "USER",
                 "PID",
                 "%CPU",
                 "%MEM",
                 "VSZ",
                 "RSS",
                 "TTY",
                 "STAT",
                 "START",
                 "TIME",
                 "COMMAND"
                 ],
         "Processes":[
                 ["root","20147","0.0","0.1","18060","1864","pts/4","S","10:06","0:00","bash"],
                 ["root","20271","0.0","0.0","4312","352","pts/4","S+","10:07","0:00","sleep","10"]
         ]
    }

But when I run the same api on my dock enabled VM, I got

    "Processes": [
        [
            "root",
            "27467",
            "1449",
            "0",
            "19:39",
            "pts/3",
            "00:00:00",
            "/bin/bash"
        ],
        [
            "root",
            "28010",
            "27467",
            "0",
            "19:46",
            "pts/3",
            "00:11:03",
            "/bin/bash"
        ]
    ],
    "Titles": [
        "UID",
        "PID",
        "PPID",
        "C",
        "STIME",
        "TTY",
        "TIME",
        "CMD"
    ]

which doesn't contain the cpu and mem %. Is there anything I need to enable?

My docker version is 
suse64:~ # docker version
Client version: 1.1.1
Client API version: 1.13
Go version (client): go1.3
Git commit (client): dc62f3c
Server version: 1.1.1
Server API version: 1.13
Go version (server): go1.3
Git commit (server): dc62f3c

Thanks a lot in advance.
Jannle

### PR DESCRIPTION
**Pull request by MHBauer**
- update example based on '-ef' behavior
- new example specifying ps args 'aux'
- resolves #7205

### CODE REVIEW

**Comment by MHBauer** [comment]
attn: @thaJeztah @enlinxu @moxiegirl

**Comment by thaJeztah** [comment]
Thanks @MHBauer, LGTM

**Comment by MHBauer** [comment]
Should I update the older versions of the api as well? I am not sure when the behavior changed (though I can go check, if it is worth the time).

**Comment by thaJeztah** [comment]
@MHBauer that'd be great! (was thinking of that when reviewing, but it's not a show stopper)

**Comment by MHBauer** [comment]
I can see it going back to v1.2.0 (API v1.14). Updated all. Very boring, but please check that they all look exactly the same...

**Comment by thaJeztah** [comment]
Awesome! LGTM

ping @SvenDowideit @moxiegirl @jamtur01

File: docs/reference/api/docker_remote_api_v1.14.md [hunk 1]
```diff
@@ -269,40 +269,59 @@ Status Codes:
 
 `GET /containers/(id)/top`
 
-List processes running inside the container `id`
+List processes running inside the container `id`. On unix systems this
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.14.md]: Unix

File: docs/reference/api/docker_remote_api_v1.14.md [hunk 2]
```diff
@@ -269,40 +269,59 @@ Status Codes:
...
+is done by running the system `ps` command. This endpoint is not
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.14.md]: -system

File: docs/reference/api/docker_remote_api_v1.14.md [hunk 3]
```diff
@@ -269,40 +269,59 @@ Status Codes:
...
+        [
+          "root","13895","0.0","0.0","4348","692","pts/0","S+","17:15","0:00","sleep 10"
+        ]
+      ],
+    }
 
 Query Parameters:
 
--   **ps_args** – ps arguments to use (e.g., aux)
+-   **ps_args** – ps arguments to use (e.g., `aux`), defaults to `-ef`
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.14.md]: \`ps\`

**Comment by jamtur01** [comment]
Minor suggestions otherwise LGTM.

**Comment by MHBauer** [comment]
Changes handled.

**Comment by thaJeztah** [comment]
Thanks, re-LGTM!!

**Pull request merged**